raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -2135,66 +1746,34 @@ def test_mediator_lock_expired_after_receive_secret_reveal():
     - M should remove the lock from both secrethashes_to_lockedlocks
       and secrethashes_to_unlocklocks.
     """
-    amount = 10
     block_number = 5
-    target = HOP2
     expiration = 30
     pseudo_random_generator = random.Random()
 
-    payer_channel = factories.make_channel(
-        partner_balance=amount,
-        partner_address=UNIT_TRANSFER_SENDER,
-        token_address=UNIT_TOKEN_ADDRESS,
-    )
-
-    payer_transfer = factories.make_signed_transfer_for(
-        payer_channel,
-        amount,
-        UNIT_TRANSFER_SENDER,
-        target,
-        expiration,
-        UNIT_SECRET,
-    )
-
-    payee_channel = factories.make_channel(
-        our_balance=amount,
-        token_address=UNIT_TOKEN_ADDRESS,
-        partner_address=target,
-    )
-    channel_map = {
-        payee_channel.identifier: payee_channel,
-        payer_channel.identifier: payer_channel,
-    }
-    possible_routes = [factories.route_from_channel(payee_channel)]
-
-    init_mediator = ActionInitMediator(
-        possible_routes,
-        factories.route_from_channel(payer_channel),
-        payer_transfer,
-    )
+    channels = mediator_make_channel_pair()
```

Comment:
It was specified since `HOP2` was always used as target in multiple tests. I changed the default target to `UNIT_TRANSFER_TARGET` now to make it more clear.